### ISSUE TITLE
mod2gbt/mod2gbt: Permission denied

### ISSUE THREAD

**Issue by ThomasFoydel** [chrismaltby/gb-studio#655]
When I try to run, I get the error: 
```
Preparing variables...
Preparing strings...
Preparing images...
Preparing ui...
Preparing sprites...
Preparing avatars...
Preparing music...
Preparing scenes...
Preparation complete
Compiling events...
Unlink _gbsbuild
Copy default engine
Looking for local engine in assets/engine
Local engine not found, using default engine
Generate header: data_ptrs.h
Generate data file: data_ptrs.c
Generate header: banks.h
Generate data file: bank_6.c
Generate data file: bank_7.c
Generate data file: bank_8.c
Generate data file: bank_9.c
Generate data file: bank_10.c
Generate object file: bank_6.o
Generate object file: bank_7.o
Generate object file: bank_8.o
Generate object file: bank_9.o
Generate object file: bank_10.o
mod2gbt "/Users/thomasfoydel/Documents/gameboy/Untitled/assets/music/Rulz_FastPaceSpeedRace.mod" music_track_101_ 8
/bin/sh: /var/folders/f7/cw9n_bcn52l4j4jk7rksc09r0000gn/T/_gbstools/mod2gbt/mod2gbt: Permission denied
126

Warnings:
- /bin/sh: /var/folders/f7/cw9n_bcn52l4j4jk7rksc09r0000gn/T/_gbstools/mod2gbt/mod2gbt: Permission denied
- 126
```
**To Reproduce**
I previously had gbs1 installed, I only opened it a couple times and changed the music file. I uninstalled gbs1 and installed the new gbs. The first few times I ran it the demo worked fine, now I get this error message if I try to run the demo. If I try to run a new blank project with a new blank scene it produces a similar error. 

I tried uninstalling and deleting all project files, then downloading again and running gbs 2.0.0-beta4 and I get the same result

**Platform (please complete the following information):**
 - OS: macOS 10.14.
 - App version 2.0.0-beta4

**Comment by RichardULZ** [comment]
Have you rebooted your computer? (Said to fix up temp folder stuff on Mac sometimes)
If that doesn't work, maybe try to get to the file location and delete the _gbstools folder (it should copy it across again).

There should also be a button to clear the build cache, but I think it's gotten stuck a bit deeper than that.

Thanks for the report, I've never seen mod2gbt specifically give a permission denied error, but there was another Mac related temp/perms issue reported recently, I think that was closed.

**Comment by ThomasFoydel** [comment]
I tried clearing the cache but that did not work, restarting the computer did, so that worked, or the combination of clearing the cache and restarting worked, so I guess it was a temp file thing. Can't believe I forgot the tried and true 'turn it off and on again.' 
I'm not sure what I changed that broke it, but I think it is from when I was messing with the music in a previous version. I downloaded some mod files of the tetris theme and replaced the template.mod file with one of them in a project, though I changed it back after, and this problem was happening in any/all projects even brand new sample and blank projects.
Thank you for advising, have a great day.
